charts/risingwave/templates/etcd-secret.yaml

```diff
@@ -19,3 +20,4 @@ stringData:
   RW_ETCD_USERNAME: {{ .Values.metaStore.etcd.authentication.username | quote }}
   RW_ETCD_PASSWORD: {{ .Values.metaStore.etcd.authentication.password | quote }}
 {{- end }}
+{{- end }}
```

Comment:
nit: new line

Reply:
Append them once for all in https://github.com/risingwavelabs/helm-charts/pull/36